vllm/model_executor/layers/attention.py

```diff
@@ -85,7 +87,9 @@ def set_attn_bias(
             # Already set by a previous layer.
             return
         prompt_lens = input_metadata.prompt_lens
-        attn_bias = BlockDiagonalCausalMask.from_seqlens(prompt_lens)
+        attn_bias = BlockDiagonalMask.from_seqlens(prompt_lens)
```

Comment:
@timlacroix Shouldn't this be `BlockDiagonalCausalMask`?

Reply:
you're right. I'm tired, sorry. fixed.